src/gateway/production/logging/update-log-level-dynamically.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Dynamic Log Level Updates
+content-type: reference
+---
+
+
+With {{site.base_gateway}} 3.1, you can now change the log level of {{site.base_gateway}} dynamically, without restarting {{site.base_gateway}} using the Admin API. This set of endpoints can be protected using [RBAC](/gateway/latest/admin-api/rbac/reference/#add-a-role-endpoint-permission) and changes in log level are reflected in the [audit log](/gateway/latest/kong-enterprise/audit-log/).
```

Comment:
```suggestion
With {{site.base_gateway}} 3.1, you can now change the log level of {{site.base_gateway}} dynamically, without restarting {{site.base_gateway}} using the Admin API. This set of endpoints can be protected using [RBAC](/gateway/latest/admin-api/rbac/reference/#add-a-role-endpoint-permission) and changes in log level are reflected in the [audit log](/gateway/latest/kong-enterprise/audit-log/).

The log level change is propagated to all Nginx workers of a node, including to newly spawned workers.

Care must be taken when changing the log level of a node to `debug` in a production environment because the disk could fill out quickly. As soon as the debug logging finishes, ensure to revert back to a higher level such as `notice`.
```